src/provider.ts

```diff
@@ -112,7 +114,7 @@ export class DelphinusBrowserConnector extends DelphinusProvider<BrowserProvider
     );
   }
 
-  async switchNet(chainHexId: string) {
+  async switchNet(chainHexId: string, networkOptions?: AddNetworkOptions) {
```

Comment:
networkOptions is ok.
Not sure name it as networkOptions or networkParams. (AddNetworkOptions or AddNetworkParams)